app.js

```diff
@@ -0,0 +1,17 @@
+const express = require("express");
+
+const { getTopics } = require("./controllers/nc_news.controller");
+
+const {
+	serverErrorsHandler,
+	handleNotFound,
+} = require("./controllers/error.controller");
+
+const app = express();
+
+app.get("/api/topics", getTopics);
+
+app.use(handleNotFound);
```

Comment:
I would expect the all bad urls to be handled at the end of our express chain with an app.all()